modules/Lending.py

```diff
@@ -242,7 +244,8 @@ def lend_all():
             break
     try:
         for cur in lending_balances:
-            usable_currencies += lend_cur(cur, total_lent, lending_balances, ticker)
+            if cur in all_currencies and cur not in blacklist_currencies:
```

Comment:
As I will say in my final review, a whitelist system with `all_currencies` is sufficient for determining if a coin is to be lent.

Reply:
Yep, I had problems with all_currencies. Maybe, I did a mistake at this point in time. So, I will use it again to look if it is working, now.